app/plugins/shorten-url/index.js

```diff
@@ -0,0 +1,47 @@
+const browser = require('webextension-polyfill');
+const $ = require('jquery');
```

Comment:
We can probably do this without jQuery and use vanilla javascript so we don't add another dependency for making an AJAX request